ospfd/ospf_vty.c

```diff
@@ -4440,22 +4451,41 @@ static void show_ip_ospf_neighbor_sub(struct vty *vty,
 					json_object_string_add(json_neighbor,
 							       "deadTimeMsecs",
 							       "inactive");
+					json_object_string_add(
+						json_neighbor,
+						"routerDeadIntervalTimerDueMsec",
```

Comment:
nit: I feel, key could be 'routerDeadTimerIntervalDueMsec'

Reply:
I'm just keeping the same naming as it's already in this file.